.yarn/patches/@patternfly-pfe-core-npm-2.0.0-next.1-e0ea84a032

```diff
@@ -0,0 +1,113 @@
+diff --git a/package.json b/package.json
+index 50076e841533b3154e6063de906d86ed823da705..604728d1d5f666f92da7c15993dd50cdecf6d31e 100644
+--- a/package.json
```

Comment:
https://www.npmjs.com/package/@patternfly/pfe-core

`@patternfly/pfe-core@2.0.0-next.2` should fix this